### ISSUE TITLE
Error during VM creation

### ISSUE THREAD

**Issue by gfkauer** [Telmate/terraform-provider-proxmox#1140]
```
Plan: 1 to add, 0 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

proxmox_vm_qemu.vm_example: Creating...
...
...
...
proxmox_vm_qemu.vm_example: Still creating... [7m50s elapsed]
╷
│ Error: Plugin did not respond
│ 
│   with proxmox_vm_qemu.vm_example,
│   on main.tf line 17, in resource "proxmox_vm_qemu" "vm_example":
│   17: resource "proxmox_vm_qemu" "vm_example" {
│ 
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ApplyResourceChange call. The plugin logs may contain more details.
╵

Stack trace from the terraform-provider-proxmox_v2.9.14 plugin:

panic: interface conversion: interface {} is string, not float64

goroutine 39 [running]:
github.com/Telmate/proxmox-api-go/proxmox.NewConfigQemuFromApi(0xc00053e718, 0xc9d509?)
        github.com/Telmate/proxmox-api-go@v0.0.0-20230319185744-e7cde7198cdf/proxmox/config_qemu.go:584 +0x4605
github.com/Telmate/terraform-provider-proxmox/proxmox.resourceVmQemuCreate(0xc0000dd280, {0xb66f60?, 0xc00043caa0})
        github.com/Telmate/terraform-provider-proxmox/proxmox/resource_vm_qemu.go:972 +0x2c4d
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).create(0xdd7840?, {0xdd7840?, 0xc00040d020?}, 0xd?, {0xb66f60?, 0xc00043caa0?})
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.25.0/helper/schema/resource.go:695 +0x178
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).Apply(0xc00033c7e0, {0xdd7840, 0xc00040d020}, 0xc0000da270, 0xc0000dd100, {0xb66f60, 0xc00043caa0})
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.25.0/helper/schema/resource.go:837 +0xa85
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ApplyResourceChange(0xc0003c6b70, {0xdd7840?, 0xc00040cf00?}, 0xc0005882d0)
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.25.0/helper/schema/grpc_provider.go:1021 +0xe8d
github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ApplyResourceChange(0xc0002983c0, {0xdd7840?, 0xc00040c1b0?}, 0xc0001923f0)
        github.com/hashicorp/terraform-plugin-go@v0.14.3/tfprotov5/tf5server/server.go:818 +0x574
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ApplyResourceChange_Handler({0xc6bc20?, 0xc0002983c0}, {0xdd7840, 0xc00040c1b0}, 0xc000192000, 0x0)
        github.com/hashicorp/terraform-plugin-go@v0.14.3/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:385 +0x170
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0003de000, {0xddb420, 0xc00040e1a0}, 0xc0002e8000, 0xc0003e0c30, 0x128f7a0, 0x0)
        google.golang.org/grpc@v1.53.0/server.go:1336 +0xd23
google.golang.org/grpc.(*Server).handleStream(0xc0003de000, {0xddb420, 0xc00040e1a0}, 0xc0002e8000, 0x0)
        google.golang.org/grpc@v1.53.0/server.go:1704 +0xa2f
google.golang.org/grpc.(*Server).serveStreams.func1.2()
        google.golang.org/grpc@v1.53.0/server.go:965 +0x98
created by google.golang.org/grpc.(*Server).serveStreams.func1
        google.golang.org/grpc@v1.53.0/server.go:963 +0x28a

Error: The terraform-provider-proxmox_v2.9.14 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.
```

**Comment by Tinyblargon** [comment]
@gfkauer currently the 2.x versions are broken and the fix has breaking changes, please use 3.0.1rc4 instead.

**Comment by gfkauer** [comment]
Thanks @Tinyblargon , I'll try it and close the issue.
